web/templates/remind_me_popover_content.hbs

```diff
@@ -1,39 +0,0 @@
-{{! Contents of the "remind me about this actions" popup }}
-<ul class="nav nav-list remind_me_popover">
-    <li>
-        <a class="remind in_20m" data-message-id="{{message.id}}" tabindex="0">
-            {{t "in 20 minutes" }}
-        </a>
-    </li>
```

Comment:
This last commit message should also reference f40855bad232efe506fe53a361bcb422d1089ba2, which removed the feature this was part of.

Reply:
Referenced.